_data/simple-icons.json

```diff
@@ -6909,6 +6909,11 @@
             "source": "https://kdenlive.org/en/logo/",
             "guidelines": "https://kdenlive.org/en/logo/"
         },
+        {
+            "title": "Kedro",
+            "hex": "FFC900",
+            "source": "https://github.com/kedro-org/kedro-brand-identity/blob/main/icon/black/kedro-icon-svgo-black.svg"
+        },
```

Comment:
We could also add `https://github.com/kedro-org/kedro-brand-identity/tree/7f7b380cb1a2951c06ca292f0d2b442db895f804` for guidelines since we can access their brand guidelines and their trademark policy from it.